src/components/Extensibility/hooks/useDebugger.scss

```diff
@@ -0,0 +1,46 @@
+.debugonata {
+  width: auto;
```

Comment:
I will suggest to not make this auto, for better UX something like 70% could be better